tests/src/test/scala/common/Wsk.scala

```diff
@@ -1040,13 +1040,14 @@ trait RunWskCmd extends Matchers {
             env: Map[String, String] = Map("WSK_CONFIG_FILE" -> ""),
             workingDir: File = new File("."),
             stdinFile: Option[File] = None,
-            showCmd: Boolean = false): RunResult = {
+            showCmd: Boolean = false,
+            hideFromOutput: Seq[String] = Seq()): RunResult = {
         val args = baseCommand
         if (verbose) args += "--verbose"
-        if (showCmd) println(args.mkString(" ") + " " + params.mkString(" "))
+        if (showCmd) println(hideStr(args.mkString(" ") + " " + params.mkString(" "), hideFromOutput))
```

Comment:
Yep, in some places.  IMHO, I kind of like seeing the actual commands used within a test.  When troubleshooting a failure, it's easy to see what's going on from the logs.. and it's possible to copy/paste the command(s) from the log to manually recreate each step in the test.